src/index.jsx

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 
 import App from './components/App';
-
+import './styles/App.styl';
```

Comment:
I think `App.styl` import should be done in `App.jsx`, not in `index.jsx`. What do you think?

Reply:
you are right, will move to App component